<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisDlg</class>
 <widget class="BaseDlg" name="AnalysisDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnalysisDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>618</y>
     <width>106</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shield_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>238</y>
     <width>110</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>项目遮蔽</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">flat_button</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>81</y>
     <width>415</width>
     <height>142</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-right: 0;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="fonts" stdset="0">
    <string notr="true">20</string>
   </property>
  </widget>
  <widget class="QLabel" name="bk_start_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>710</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>41</y>
     <width>361</width>
     <height>272</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="SeqNoLab">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>起始样本号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SeqNoEdit">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-bottom: 10px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="auto_recheck_ckbox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>自动复查</string>
        </property>
        <property name="checks" stdset="0">
         <string notr="true">push</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="communication_ckbox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>主机通讯</string>
        </property>
        <property name="checks" stdset="0">
         <string notr="true">push</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>9</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="maintain_checkbox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>开始前执行</string>
      </property>
      <property name="checks" stdset="0">
       <string notr="true">push</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maxVisibleItems">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <zorder>bk_start_label</zorder>
  <zorder>OkBtn</zorder>
  <zorder>CancelBtn</zorder>
  <zorder>textEdit</zorder>
  <zorder>shield_btn</zorder>
  <zorder>widget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>maintain_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
